src/components/PanelV2/PanelV2.stories.js

```diff
@@ -39,8 +38,9 @@ const { interactive01, text04 } = theme;
 const closeButtonLabel = 'Close';
 
 const props = {
-  title: label,
-  subtitle: label,
+  title: 'Example title',
```

Comment:
This is minor, and perhaps a follow-up, but it would be worth exposing these as knobs for consumers to experiment with

Reply:
Thanks! Yeah I'll put that in a quick follow up after sprint exit 👍